models/clm/CAM_bias_correction/CAM4_site_grid.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Extracts site level grid cell from CAM4 reanalysis
+
+# CAM4 reanalysis
+CAM4_grid_path="/glade/collections/rda/data/ds199.1/"
+# Site Level grid cell from CAM4 renanalysis
+CAM4_site_path="/glade/work/bmraczka/CAM4_NR1/"
+
+# The US-NR1 flux tower site is used as an example 
+# NR1 location (40.03, -105.55) or (40.03, 254.45)
+# CAM4 grid is 1.95x2.5  or 288 longitude grids and 96 latitude grids
+# This lat/lon corresponds with doma_lon= 102 (255 degrees east) ; doma_lat= 69  ; (40.7368 degrees)
+
+
+for YEAR in {1997..2010..1}
```

Comment:
I don't use sh much.  Is this indenting standard or helpful?
I can understand indenting the 'for' and 'do' lines the same,
but the contents would be clearer if they were indented more.